File: 02python/lecture-4/pattern1.py
import os
os.system("cls")

def main():
    r=int(input("Enter number of rows r="))
    c=int(input("Enter number of columns c="))

    i=1
    j=1
    while(i<=r):
        while(j<=c):
            print(j,end="")
            j=j+1
        print(end="\n")
        j=0
        i=i+1

if __name__=="__main__":
    main()

File: Lecture-4/pattern1.cpp
#include<iostream>
#include<stdlib.h>

using namespace std;

int main(){
    system("CLS");
    int r, c;
    cout<<"enter the amount of rows and columns"<<endl;
    cin>>r>>c;
    int i=0, j=0;
    while(i<r){
        while(j<c){
            cout<<j+1;
            j++;
        }
        cout<<endl;
        j=0;
        i++;
    }
}